<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="voltage_frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>421</width>
     <height>211</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="GS610R_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>64</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>GS610R:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GS610RConnect_pushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="GS610RRange_comboBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0.2 V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12 V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20 V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30 V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>60 V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>110 V</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="GS610RAddress_lineEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="GS610LConnect_pushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="GS610LRange_comboBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0.2 V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12 V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20 V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30 V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>60 V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>110 V</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="GS610L_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>114</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>GS610L:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="GS610LAddress_lineEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="GS210_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>164</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>GS210:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GS210Connect_pushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="GS210Range_comboBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0.01 V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0.1 V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10 V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30 V</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="GS210Address_lineEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="voltageInstrument_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Instrument</string>
    </property>
   </widget>
   <widget class="QLabel" name="voltageConnect_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>67</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="voltageRange_label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>67</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Range</string>
    </property>
   </widget>
   <widget class="QLabel" name="voltageAddress_label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>67</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="Magnet_frame">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>421</width>
     <height>211</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Magnet_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>56</y>
      <width>67</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Magnet:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MagnetConnect_pushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="MagnetAddress_lineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>190</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="MagnetRate_lineEdit">
    <property name="geometry">
     <rect>
      <x>329</x>
      <y>110</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="MagnetRate_label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Specify Rate:
(T/min)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="MagnetSweepMode_comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>As Fast As Possible</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Specify Rate</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="MagnetInstrument_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Instrument</string>
    </property>
   </widget>
   <widget class="QLabel" name="MagnetConnect_label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>67</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="MagnetAddress_label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>67</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="MagnetPOC_label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Persistent On Completion:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="MagnetPOC_checkBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="exit_pushButton">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>260</y>
     <width>90</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="apply_pushButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>260</y>
     <width>90</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>GS610RConnect_pushButton</tabstop>
  <tabstop>GS610RRange_comboBox</tabstop>
  <tabstop>GS610RAddress_lineEdit</tabstop>
  <tabstop>GS610LConnect_pushButton</tabstop>
  <tabstop>GS610LRange_comboBox</tabstop>
  <tabstop>GS610LAddress_lineEdit</tabstop>
  <tabstop>GS210Connect_pushButton</tabstop>
  <tabstop>GS210Range_comboBox</tabstop>
  <tabstop>GS210Address_lineEdit</tabstop>
  <tabstop>MagnetConnect_pushButton</tabstop>
  <tabstop>MagnetAddress_lineEdit</tabstop>
  <tabstop>MagnetSweepMode_comboBox</tabstop>
  <tabstop>MagnetRate_lineEdit</tabstop>
  <tabstop>MagnetPOC_checkBox</tabstop>
  <tabstop>apply_pushButton</tabstop>
  <tabstop>exit_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
